packages/components/src/panel/body.js

```diff
@@ -50,6 +50,7 @@ class PanelBody extends Component {
 							className="components-panel__body-toggle"
 							onClick={ this.toggle }
 							aria-expanded={ isOpened }
+							aria-label={ title }
```

Comment:
#2375 is from August 2017 🙂and many things have changed since then. Specifically, this Button used to have a `label` prop that was rendered as a `label=""` HTML attribute. It was removed in https://github.com/WordPress/gutenberg/pull/2591/files#diff-db33f7945064a86aa02f468ca5cb4184L54

I'd say this `aria-label` shouldn't be added as it just duplicates the button text.

Reply:
😄

Ah, I wasn't sure if it was useful to add here so I erred on the side of caution. Thanks, I'll remove it! 👍